requirements.txt

```diff
@@ -1,7 +0,0 @@
-affine>=1.3.0
-boto3>=1.2.4
-cligj
-enum34
-numpy>=1.8
-snuggs>=1.2
-setuptools
```

Comment:
@geowurster what do you think about restoring the requirements files and dealing with that in a later PR so we don't hold up the doc changes?